app/src/editor/canvas/utils/isModuleResizable.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+const RESIZABLE_MODULES = [
+    'ChartModule',
+    'CommentModule',
+    'CustomModule',
+    'GaugeModule',
```

Comment:
`CustomModule` probably doesn't need to be resizable, it only has the one button to edit code. `GaugeModule` can be removed, I think.